src/staging/src/pages/alerts.js

```diff
@@ -138,138 +390,115 @@ const PublicAdvisoryPage = ({ data }) => {
+        </div>
+        }
+
+        <div className={ isDataOld ? classes.loadingArea : "hidden" }>
+          <div className={classes.loadingSpinner}><CircularProgress></CircularProgress></div>
+          <div className={classes.loadingText}>Loading...</div>
+        </div>
+        
+        <div className={isDataOld ? "hidden" : undefined}>
+          
+          <div className={isAnySearch ? classes.filterResult : "hidden"}>
+            {!isSearchError && <>Advisories that match your search: {filterCount}</>}
+            {isSearchError && "There was an error in your search. Tip: avoid using punctuation"}
+          </div>
+          
+          <div className={classes.legend}>
+            <div className={classes.legendItem}>
+              <div className={classes.redAlert + ' ' + classes.legendCircle}>&nbsp;</div>
+              <div className={classes.legendLevel}>High</div>
+            </div>
+            <div className={classes.legendItem}>
+              <div className={classes.yellowAlert + ' ' + classes.legendCircle}>&nbsp;</div>
+              <div className={classes.legendLevel}>Moderate</div>
+            </div>
+            <div className={classes.legendItem}>
+              <div className={classes.blueAlert + ' ' + classes.legendCircle}>&nbsp;</div>
+              <div className={classes.legendLevel}>Low</div>
+            </div>
+          </div>
+
+          <AdvisoryList advisories={advisories} pageIndex={pageIndex} pageLen={pageLen}></AdvisoryList>
+          
+          <AdvisoryPageNav pageIndex={pageIndex} pageCount={pageCount} setPage={setPage}></AdvisoryPageNav>
+          
+        </div>
+
       </Container>
+      <br /><br />
       <Footer>{data.strapiWebsites.Footer}</Footer>
     </>
   )
 }
 
 export default PublicAdvisoryPage
 
-export const query = graphql`
-  {
-    allStrapiPublicAdvisory(sort: { fields: advisoryDate, order: DESC }) {
-      nodes {
-        id
-        title
-        description
-        isAdvisoryDateDisplayed
-        isEffectiveDateDisplayed
-        isEndDateDisplayed
-        isReservationsAffected
-        isSafetyRelated
-        urgency {
-          code
-          color
-          sequence
-          urgency
-        }
-        protectedAreas {
-          orcs
-          protectedAreaName
-          hasCampfireBan
-          hasSmokingBan
-          slug
-        }
-        accessStatus {
-          color
-          accessStatus
-          precedence
-        }
-        eventType {
-          eventType
-        }
-        advisoryDate(formatString: "MMMM DD, YYYY")
-        advisoryNumber
-        dcTicketNumber
-        effectiveDate(formatString: "MMMM DD, YYYY")
-        endDate(formatString: "MMMM DD, YYYY")
-        expiryDate(formatString: "MMMM DD, YYYY")
+/*
+
+  - This section of the graphql query was removed as advisories
+  - are now being loaded through the api
+  - Leaving this here as a comment for future reference
```

Comment:
I appreciate the description but still think that this comment is better removed. We have access to old code via git, and what it used to do isn't really relevant anymore.